src/components/common/FileUploadPreview.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <v-form
     ref="form"
+    class="file-upload-preview"
```

Comment:
I moved this to the top level so that multiple instances of this are at the same level and I can use easy CSS selectors on it (such as `.file-upload-preview:not(:first-of-type)`), ie:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/6a3eb268-07f4-4a3f-bcf1-68fe22f7a911)